app/views/unattended/provisioning_templates/PXELinux/preseed_default_pxelinux_autoinstall.erb

```diff
@@ -44,8 +47,8 @@ test_on:
 #
 DEFAULT linux cloud-init autoinstall
 LABEL linux cloud-init autoinstall
-    KERNEL ubuntu/focal/vmlinuz
-    INITRD ubuntu/focal/initrd
-    APPEND ip=dhcp url=http://<%= @preseed_server %><%= @preseed_path %> autoinstall ds=nocloud-net;s=http://<%= @preseed_server %>/pub/preseed/autoinstall/<%= @host.shortname %>/ root=/dev/ram0 ramdisk_size=1500000 fsck.mode=skip
+    KERNEL <%= @kernel %>
+    INITRD <%= @initrd %>
+    APPEND ip=dhcp url=http://<%= @preseed_server %><%= image_path %> autoinstall ds=nocloud-net;s=http://<%= userdata_server %>/userdata/ root=/dev/ram0 ramdisk_size=1500000 fsck.mode=skip
```

Comment:
note for myself: `/userdata/` is handled by the `UserdataController` (`app/controllers/userdata_controller.rb`) and is proxied on the proxy with the `template` feature, *good*.